sx127x/sx127x.go

```diff
@@ -244,9 +270,9 @@ func (d *Device) SetLowDataRateOptim(val uint8) {
 // SetLowFrequencyModeOn enables Low Data Rate Optimization
 func (d *Device) SetLowFrequencyModeOn(val bool) {
 	if val {
-		d.WriteRegister(SX127X_REG_OP_MODE, d.ReadRegister(SX127X_REG_OP_MODE)|0x04)
+		d.WriteRegister(SX127X_REG_OP_MODE, d.ReadRegister(SX127X_REG_OP_MODE)|(1<<SX127X_OPMODE_LOW_FREQUENCY_Pos))
```

Comment:
We could also define `SX127X_OPMODE_LOW_FREQUENCY` constant and set it to `0x04` and then just do:
```suggestion
		d.WriteRegister(SX127X_REG_OP_MODE, d.ReadRegister(SX127X_REG_OP_MODE)|SX127X_OPMODE_LOW_FREQUENCY_BIT)
```